tests/idoverridegroup/test_idoverridegroup.yml

```diff
@@ -0,0 +1,208 @@
+---
+- name: Test idoverridegroup
+  hosts: "{{ ipa_test_host | default('ipaserver') }}"
+  # It is normally not needed to set "become" to "true" for a module test.
+  # Only set it to true if it is needed to execute commands as root.
```

Comment:
These comments should be removed.

Maybe we should state that in the templates?

Reply:
Fixed.